model/json/model.go

```diff
@@ -85,8 +86,9 @@ type Reference struct {
 
 // Process is the process emitting a set of spans
 type Process struct {
-	ServiceName string     `json:"serviceName"`
-	Tags        []KeyValue `json:"tags"`
+	ServiceName string            `json:"serviceName"`
+	Tags        []KeyValue        `json:"tags"`
+	TagsMap     map[string]string `json:"tagsMap,omitempty"`
```

Comment:
I'm not a maintainer, but if it was me, I would have a separate "span" model that is independent of the backend and then I would map it to a backend specific model in the backend specific package. It does not look right to me that you have to modify the common model just to change ES representation.